scripts/docker-build.sh

```diff
@@ -0,0 +1,36 @@
+#! /bin/sh
+
+PLATFORM=linux/amd64
+# We can add linux/arm64 back in if we need it
+REGION='eu-west-2'
+
+while [ $# -gt 0 ]; do
+  if [[ $1 == *"--"* ]]; then
+    param="${1/--/}"
+    declare $param="$2"
+  fi
+  shift
+done
+
+if [[ $ENVIRONMENT == "development" ]]; then
+  ECS_ACCOUNT_NUMBER=388086622185
```

Comment:
Should we be hard coding the account numbers in this repo like this rather than fetching them from somewhere else?